### ISSUE TITLE
New Verified Projects Tweet Thread R22

### ISSUE THREAD

**Issue by NikolaCreatrix** [Giveth/giveth-planning#943]
Creating this ongoing issue based on the last comms call discussion.

To start I am adding a link to the current tweet thread for R21 here in the issue #931 which is ready for review.

For the next rounds, I will compile all previous tweet threads into one document to have one place of reference for all the previous rounds.

With the next one, I will just post a draft here and tag 'ready for review' once its ready for review. ;)

**Comment by laurenluz** [comment]
@NikolaCreatrix are we still using this issue? did we make a thread about Rd23?

I find this issue/flow confusing and would much prefer an new issue for each thread.

**Comment by laurenluz** [comment]
imo, it makes a mess of things to use one issue for many different documents. it gets too long and hard to follow, and feels cumbersome to never close the issue and see it move back and forth all the time.

I prefer to have more specific issues - one for each round in this case - that get closed once they are posted.

You can already see that this issue is getting long with comments. The better way to keep track of multiple issues like this is under an epic... where you can see all closed issues & still open issues, so anyone who wants to see past data can find it through there.

Anyway, my opinion... if you & @c0ric0ri I'll try to keep track of this.

And cool re: the draft! Tag me when you're ready :)

**Comment by NikolaCreatrix** [comment]
Hola hey, slow but steady I'm ready. :)

https://docs.google.com/document/d/1VB-D-KbHoMyT_SAMjbuzSvpAleLL_iWI7A0wwBJs3-Y/edit?usp=sharing 

Ready for your feedback @laurenluz @c0ric0ri 

I compiled all the previous rounds in one document. So new round always starts at the top of the doc and rounds are divided by bold lines.

**Comment by c0ric0ri** [comment]
Hey @NikolaCreatrix great job on this! I gave it a review.

You know, what we can do is keep working through separate issues for each tweet, but always draft through the same document with the latest draft always at the top like you did this time.

So we can close this issue once the tweet is out, and start a new one with the same doc for the next round. And keep linking them to the Recurring Content epic please.

**Comment by NikolaCreatrix** [comment]
Thanks, @c0ric0ri , and thanks for your feedback in the doc. Reviewed and should be good to go.

Yes to your suggestions above. Sounds good.

**Comment by laurenluz** [comment]
Great job @NikolaCreatrix - and thanks for editing @c0ric0ri 

I made some tweaks in tweethunter... but also put some comments in the doc to keep in mind for next time.

https://twitter.com/Givethio/status/1590707539032903687
